app/presenters/topical_event_about_page_presenter.rb

```diff
@@ -0,0 +1,38 @@
+class TopicalEventAboutPagePresenter < ContentItemPresenter
+  include ExtractsHeadings
+  include ActionView::Helpers::UrlHelper
+
+  def body
+    content_item["details"]["body"]
+  end
+
+  def contents
+    extract_headings_with_ids(body).map do |heading|
+      link_to(heading[:text], "##{heading[:id]}")
+    end
+  end
+
+  def breadcrumbs
+    parent = topical_event
+    title = archived_topical_event? ? "#{parent['title']} (Archived)" : parent["title"]
+
+    [
+      {title: "Home", url: "/"},
```

Comment:
Oh: https://github.com/alphagov/govuk-lint/blob/05da1e32636f7fd695a2488d486ad78b8b7ad113/configs/rubocop/other-style.yml#L63

Maybe I'll change it and see what people think...